src/Uno.UI/Extensions/TimelineExtensions.iOS.cs

```diff
@@ -18,11 +18,14 @@ internal static void SetValueBypassPropagation(this Timeline timeline, object va
 
 			var animatedItem = timeline.PropertyInfo.GetPathItems().Last();
 
-			using (
+			var dc = animatedItem.DataContext;
+			using (dc != null ?
 				DependencyObjectStore.BypassPropagation(
-					(DependencyObject)animatedItem.DataContext,
-					DependencyProperty.GetProperty(animatedItem.DataContext.GetType(), animatedItem.PropertyName)
-				)
+					(DependencyObject)dc,
```

Comment:
It would probably be easier to read if the ternary were to be outside of the using.